src/main/java/dk/cngroup/lentils/service/ProgressService.java

```diff
@@ -1,35 +1,55 @@
 package dk.cngroup.lentils.service;
 
 import dk.cngroup.lentils.entity.Cypher;
+import dk.cngroup.lentils.entity.Status;
 import dk.cngroup.lentils.entity.Team;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 @Service
 public class ProgressService {
 
     private final StatusService statusService;
+    private final TeamService teamService;
 
     @Autowired
-    public ProgressService(StatusService statusService) {
+    public ProgressService(StatusService statusService, TeamService teamService) {
         this.statusService = statusService;
+        this.teamService = teamService;
     }
 
-    public CypherStatus setSkipProgressStatus(Cypher cypher, Team team) {
+    public void setSkipProgressStatus(Cypher cypher, Team team) {
         statusService.initializeStatusForTeamAndCypher(cypher, team);
```

Comment:
Tato inicislizace neni dobre, ale zalozil bych na to issue na githubu a vyresil to spolecne s tim "start game". Jinade se to inicializovat nebude.

Reply:
vytvorim na to issue - prozatim nechavam v kodu